mesh/mesh.sh

```diff
@@ -76,11 +77,11 @@ then
   inbox
 elif [ "$1" = "send" ]
 then
-  send "$2"
+  send "$3"
 elif [ "$1" = "download" ]
 then
-  download "$2"
+  download "$3"
 elif [ "$1" = "ack" ]
 then
-  ack "$2"
+  ack "$3"
```

Comment:
What is the purpose of $1 and $3?

Reply:
https://tecadmin.net/tutorial/bash-scripting/bash-command-arguments/